runtime/vm/jvminit.c

```diff
@@ -2072,6 +2073,8 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			if (NULL == (vm->jniWeakGlobalReferences = pool_new(sizeof(UDATA), 0, 0, POOL_NO_ZERO, J9_GET_CALLSITE(), J9MEM_CATEGORY_JNI, POOL_FOR_PORT(vm->portLibrary))))
 				goto _error;
 
+			create_and_allocate_jvm_image(vm);
```

Comment:
this should return something on failure

Reply:
we should have some error handling for this. We'll put a task for this and pr it next week